### DISCUSSION TITLE
defining a11y on homepage

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#26] [Content (Open Discussion)]
I would like to add the definition of `a11y` on the homepage, very similar if not the same way the [a11y project](https://www.a11yproject.com) has it on their site.

What should we do for the French version:

1. Should we use `a11y` and define it in French text using?
2. Should we use the less popular `a11é` and define it fully in French?
3. Something else?

**Comment by arigaud-ca** [comment]
I like a11é and it's sometimes used in France. It might not been used a lot yet but I have the felling that like t12t (in swedish) or a11d (in spanish) it could become a trend!
I would still use it with Accessibilité as weel as it's not well known.

**Comment by shawnthompson** [comment]
Add something similar to what the [a11y project](https://www.a11yproject.com) did on their page to describe the meaning of **a11y**. 

![image](https://user-images.githubusercontent.com/2575266/136412596-37b32aa6-deae-4210-907a-c55391dfefc0.png)

Image description quoted below:

> a11y stands for
ACCESSIBILITY
The phrase '11 characters'. Arrows are pointing to the first letter c and the letter t in the word 'accessibility' to show how the numeronym is formed.
The A11Y Project is a community-driven effort to make digital accessibility easier.

**Reply by SylvieG-smg** [reply]
I like this image and the idea of using "a11é" for the French, it's similar enough it should be easy to see it in either languages and make the connection for the meaning.
